pkg/network/protocols/postgres/model_linux.go

```diff
@@ -60,29 +61,27 @@ func (e *EventWrapper) Operation() Operation {
 	return e.operation
 }
 
-var (
-	regexMapper = map[Operation]*regexp.Regexp{
-		CreateTableOP: regexp.MustCompile(`CREATE TABLE (\S+)`),
-		InsertOP:      regexp.MustCompile(`INSERT INTO (\S+)`),
-		DropTableOP:   regexp.MustCompile(`DROP TABLE(\s*IF\s+EXISTS\s+)?(\S+)`),
-		UpdateOP:      regexp.MustCompile(`UPDATE (\S+)`),
-		SelectOP:      regexp.MustCompile(`SELECT .* FROM (\S+)`),
-	}
-)
-
 // extractTableName extracts the table name from the query.
 func (e *EventWrapper) extractTableName() string {
-	regex, ok := regexMapper[e.Operation()]
-	if ok {
-		if matches := regex.FindSubmatch(e.Tx.getFragment()); len(matches) > 1 {
-			return string(
-				bytes.Trim( // Remove leading and trailing quotes from the table name
-					bytes.ReplaceAll(matches[len(matches)-1], []byte{0}, []byte{}), // Remove null bytes
-					"\""),
-			)
-		}
+	fragment := string(e.Tx.getFragment())
+	// Check if the string contains "IF EXISTS",
+	// temp solution for the fact that ObfuscateSQLString does not support "IF EXISTS".
+	if strings.Contains(fragment, "IF EXISTS") {
+		fragment = strings.ReplaceAll(fragment, "IF EXISTS", "")
+	}
```

Comment:
you can use `bytes.Contains` and `bytes.ReplaceAll` instead of converting to string and performing string operations

Reply:
mmm I still need to convert it to string in the ObfuscateSQLString function. And when I use the bytes I would need to convert the "IF EXISTS" to byte which feels the same for me